src/domain/common/whiteboard/dto/whiteboard.dto.create.ts

```diff
@@ -1,11 +1,24 @@
 import { Field, InputType, ObjectType } from '@nestjs/graphql';
-import { IsOptional } from 'class-validator';
+import { IsOptional, ValidateNested } from 'class-validator';
 import { WhiteboardContent } from '@domain/common/scalars/scalar.whiteboard.content';
-import { CreateNameableInput } from '@domain/common/entity/nameable-entity/dto/nameable.dto.create';
+import { CreateProfileInput } from '@domain/common/profile/dto';
+import { Type } from 'class-transformer';
+import { NameID } from '@domain/common/scalars';
 
 @InputType()
 @ObjectType('CreateWhiteboardData')
-export class CreateWhiteboardInput extends CreateNameableInput {
+export class CreateWhiteboardInput {
+  @Field(() => CreateProfileInput, { nullable: true })
```

Comment:
the thing is `CreateNameableInput` is an `@InputType` but not an `@ObjectType`, so GraphQL was not returning the profile information, only the content field:
![image](https://github.com/user-attachments/assets/07d3d11e-ed14-4a06-8140-222c284c0e9d)
![image](https://github.com/user-attachments/assets/aa5aa5c6-02cd-4139-beef-264892da3ce8)

I saw that in framing for example it's made like this: `CreateCalloutFramingData` has `profile` instead of inheriting from NameableInput:
![image](https://github.com/user-attachments/assets/20beef09-8dab-4b9d-9371-d559631d0515)

But for the whiteboard we don't have that profile available so I couldn't push the visuals into that profile when creating the whiteboard of the framing of the callout:
![image](https://github.com/user-attachments/assets/bc3fe9e4-9ca2-4dae-9379-bdc8567cf717)